--- configure.ac.bak    2021-05-29 00:06:47.000000000 -0500
+++ configure.ac        2021-05-29 00:07:39.000000000 -0500
@@ -55,6 +55,9 @@
     ,
     dnl NO
     [AC_MSG_RESULT([no])]
+    ,
+    dnl cross compiling
+    [AC_MSG_RESULT([no])]
 )
 
 AC_MSG_CHECKING([whether setjmp() saves the signal mask])
@@ -84,6 +87,9 @@
     ,
     dnl NO
     [AC_MSG_RESULT([no])]
+    ,
+    dnl cross compiling
+    [AC_MSG_RESULT([yes])]
 )
 
 AC_MSG_CHECKING([for GNU libc])
