diff --git a/src/cysignals/tests.pyx b/src/cysignals/tests.pyx
index 1a3ba52..e46041e 100644
--- a/src/cysignals/tests.pyx
+++ b/src/cysignals/tests.pyx
@@ -124,7 +124,7 @@ class return_exception(object):
         ... def raise_interrupt():
         ...     raise KeyboardInterrupt("just testing")
         >>> raise_interrupt()
-        KeyboardInterrupt('just testing',)
+        KeyboardInterrupt('just testing')
 
     """
     def __init__ (self, func):
@@ -381,7 +381,7 @@ def test_sig_error():
 
         >>> from cysignals.tests import *
         >>> test_sig_error()
-        ValueError('some error',)
+        ValueError('some error')
 
     """
     sig_on()
