From 9b3b7634dcbe294cf783d3efec1f9f8651c5ed2c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Wed, 1 Dec 2021 13:38:43 -0600
Subject: [PATCH] Fix doctest output for qemu which prints an extra line

---
 src/cysignals/tests.pyx | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/src/cysignals/tests.pyx b/src/cysignals/tests.pyx
index 0673ffa..4dbffb0 100644
--- a/src/cysignals/tests.pyx
+++ b/src/cysignals/tests.pyx
@@ -613,7 +613,7 @@ def unguarded_dereference_null_pointer():
         This probably occurred because a *compiled* module has a bug
         in it and is not properly wrapped with sig_on(), sig_off().
         Python will now terminate.
-        ------------------------------------------------------------------------
+        ...
 
     The same but with ``CYSIGNALS_CRASH_QUIET`` set. This will crash
     Python silently::
@@ -656,7 +656,7 @@ def unguarded_abort():
         This probably occurred because a *compiled* module has a bug
         in it and is not properly wrapped with sig_on(), sig_off().
         Python will now terminate.
-        ------------------------------------------------------------------------
+        ...
 
     """
     with nogil:
@@ -691,7 +691,7 @@ def unguarded_stack_overflow():
         This probably occurred because a *compiled* module has a bug
         in it and is not properly wrapped with sig_on(), sig_off().
         Python will now terminate.
-        ------------------------------------------------------------------------
+        ...
 
     """
     with nogil:
@@ -734,7 +734,7 @@ def test_bad_str(long delay=DEFAULT_DELAY):
         This probably occurred because a *compiled* module has a bug
         in it and is not properly wrapped with sig_on(), sig_off().
         Python will now terminate.
-        ------------------------------------------------------------------------
+        ...
 
     """
     cdef char* s = <char*>(16)
diff --git a/src/cysignals/tests.pyx b/src/cysignals/tests.pyx
index 0673ffa..6a8156d 100644
--- a/src/cysignals/tests.pyx
+++ b/src/cysignals/tests.pyx
@@ -615,14 +615,6 @@ def unguarded_dereference_null_pointer():
         Python will now terminate.
         ------------------------------------------------------------------------

-    The same but with ``CYSIGNALS_CRASH_QUIET`` set. This will crash
-    Python silently::
-
-        >>> import os
-        >>> env = dict(os.environ)
-        >>> env["CYSIGNALS_CRASH_QUIET"] = ""
-        >>> subpython_err('from cysignals.tests import *; unguarded_dereference_null_pointer()', env=env)
-
     """
     with nogil:
         dereference_null_pointer()
--
2.34.1
