From 64849eec02179c27fd212cbe1824ea857b1b4873 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Wed, 1 Dec 2021 16:36:15 -0600
Subject: [PATCH] remove setrlimit on macOS

---
 rundoctests.py | 14 --------------
 1 file changed, 14 deletions(-)

diff --git a/rundoctests.py b/rundoctests.py
index 8db318a..c7157a5 100755
--- a/rundoctests.py
+++ b/rundoctests.py
@@ -57,20 +57,6 @@ parser = CysignalsDocTestParser()
 print(f"Doctesting {len(filenames)} files.")
 
 
-if os.name != 'nt':
-    import resource
-    # Limit stack size to avoid errors in stack overflow doctest
-    stacksize = 1 << 20
-    if sys.platform != 'darwin':
-        # Work around a very strange OS X.
-        # This was discovered at https://github.com/sagemath/cysignals/issues/71.
-        # The original solution did not last very long and the issue reappeared.
-        resource.setrlimit(resource.RLIMIT_STACK, (stacksize, stacksize))
-
-    # Disable core dumps
-    resource.setrlimit(resource.RLIMIT_CORE, (0, 0))
-
-
 def testfile(file):
     # Child process
     try:
-- 
2.34.1

