From 9eb146b00f84a03c30b877be9facaa549bc0d5f6 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Julian=20R=C3=BCth?= <julian.rueth@fsfe.org>
Date: Wed, 1 Dec 2021 20:09:29 -0600
Subject: [PATCH] Be more verbose when running tests

---
 rundoctests.py | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/rundoctests.py b/rundoctests.py
index c7157a5..4ca86cb 100755
--- a/rundoctests.py
+++ b/rundoctests.py
@@ -63,13 +63,15 @@ def testfile(file):
         if sys.platform == 'darwin':
             from cysignals.signals import _setup_alt_stack
             _setup_alt_stack()
-        failures, _ = doctest.testfile(file, module_relative=False,
+        failures, _ = doctest.testfile(file, module_relative=False, verbose=True,
                                        optionflags=flags, parser=parser)
         if not failures:
+            print(f"Test {file} passed.")
             os._exit(0)
     except BaseException as E:
-        print(E)
+        print(f"Test {file} failed with a {type(E)} error: {E}")
     finally:
+        print(f"Test {file} failed.")
         os._exit(23)
 
 
-- 
2.34.1

